### PR TITLE
feature added userRepository

### PR DESCRIPTION
**Pull request by LukaszSzetela**

### CODE REVIEW

**Review by JanRejnowski — APPROVED** [review]

**Comment by snafx** [comment]
daje okejke

**Review by snafx — APPROVED** [review]
daje okejke!

**Review by MikolajKl — APPROVED** [review]

File: src/main/java/com/sda/Warehouse/models/User.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Email;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+@Entity(name = "users")
+@Data
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/models/User.java]: Why this model is mutable? Do you need Setters?

Comment (LukaszSzetela) [reply src/main/java/com/sda/Warehouse/models/User.java]: Ok, I will change it and I will add setter only for field isActive

**Review by snafx — APPROVED** [review]
looks good m8

**Review by RafalCiolek — APPROVED** [review]

**Pull request merged**